docs/api/alerting/unmute_alert.asciidoc

```diff
@@ -1,42 +1,52 @@
 [[unmute-alert-api]]
-=== Unmute alert API
+== Unmute alert API
 ++++
 <titleabbrev>Unmute alert</titleabbrev>
 ++++
 
 Unmute an alert.
 
 [[unmute-alert-api-request]]
-==== Request
+=== {api-request-title}
 
 `POST <kibana host>:<port>/api/alerting/rule/<rule_id>/alert/<alert_id>/_unmute`
 
 `POST <kibana host>:<port>/s/<space_id>/api/alerting/rule/<rule_id>/alert/<alert_id>/_unmute`
 
-[[unmute-alert-api-path-params]]
-==== Path parameters
+=== {api-prereq-title}
 
-`rule_id`::
-  (Required, string) The ID of the rule whose alert you want to mute.
+You must have `all` privileges for the appropriate {kib} features, depending on
+the `consumer` and `rule_type_id` of the rules you're creating. For example, the
+*Management* > *Stack Rules* feature, *Analytics* > *Discover* and *{ml-app}*
+features, *{observability}*, and *Security* features. If the rule has `actions`,
+you must also have `read` privileges for the *Management* >
+*Actions and Connectors* feature. For more details, refer to
+<<kibana-feature-privileges>>.
+
+[[unmute-alert-api-path-params]]
+=== {api-path-parms-title}
 
 `alert_id`::
```

Comment:
I noticed that you can see a list of the "muted_alert_ids" in the response from the get rule API. If that's the simplest method to find the alert IDs, I can add that info here.